components/omnibox/browser/brave_search_provider.cc

```diff
@@ -19,3 +53,21 @@ void BraveSearchProvider::DoHistoryQuery(bool minimal_changes) {
 
   SearchProvider::DoHistoryQuery(minimal_changes);
 }
+
+bool BraveSearchProvider::IsQueryPotentiallyPrivate() const {
+  if (SearchProvider::IsQueryPotentiallyPrivate()) {
+    return true;
+  }
+
+#if !BUILDFLAG(IS_IOS)
+  if (!input_.text().empty() && GetClipboardText() == input_.text()) {
+    // We don't want to send username/pwd in clipboard to suggest server
+    // accidently.
```

Comment:
Is this comment accurate?

Reply:
I think so. When user has private info in clipboard and it's copied in omnibox accidently, we want to block sending it to suggest server.